device-types/Infoblox/IB-825.yaml

```diff
@@ -0,0 +1,24 @@
+---
+manufacturer: Infoblox
+model: IB-825
+slug: infoblox-ib-825
+is_full_depth: false
```

Comment:
Hello, thank you for your contribution! In an attempt to have the most accurate and complete data we can, I have a few additional data points we would like you to add if possible.

1. Please add the `airflow` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not. I believe the airflow is `front-to-rear`
2. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not. I believe the weight is `7.71 kg`
3. Please add the manufacturer `part_number` which I believe is `TE-825`

If you have any questions, please let us know :)

Reply:
added requested fields.